scipy/interpolate/polyint.py

```diff
@@ -439,6 +439,25 @@ def approximate_taylor_polynomial(f,x,degree,scale,order=None):
     Choosing order somewhat larger than degree may improve the higher-order
     terms.
 
+    Examples
+    --------
+    We can calculate Taylor approximation polynomials of sin function with
+    various degrees:
+
+    >>> import matplotlib.pyplot as plt
+    >>> from scipy.interpolate import approximate_taylor_polynomial
+    >>> x = np.linspace(-10.0, 10.0, num=100)
+    >>> plt.plot(x, np.sin(x), label="sin curve")
+    >>> for degree in np.arange(1, 15, step=2):
+    ...     sin_taylor = approximate_taylor_polynomial(np.sin, 0, degree, 1,
+    ...                                                 order=degree + 2)
```

Comment:
Nitpick: align the star of the continuation just after the opening `(` of the previous line.
```suggestion
    ...                                                order=degree + 2)
```

Reply:
Argh, the suggested change looked much better when I previewed it.  Now the margins that github is using to display the suggested change are different, and it is hard to read the diff of the suggestion.

Just move `order=degree + 2)` to the left by one character.